tests/zfs-tests/include/libtest.shlib

```diff
@@ -3331,6 +3331,38 @@ function wait_replacing #pool
 }
 
 #
+# Wait for drives to enter a faulted state
+#
+# $1 pool name
+#
+function wait_faulted #pool
+{
+	typeset pool=${1:-$TESTPOOL}
+	while true; do
+		[[ "" != "$(zpool status $pool |
+		    grep 'FAULTED')" ]] && break
+		log_must sleep 1
+	done
+}
+
+
+#
+# Wait for drives to enter a degraded state
+#
+# $1 pool name
+#
+function wait_degraded #pool
+{
+	set -x
+	typeset pool=${1:-$TESTPOOL}
+	while true; do
+		[[ "" != "$(zpool status $pool |
+		    grep 'DEGRADED')" ]] && break
```

Comment:
This fits on 1 line under and is still under the 80 character limit.

Reply:
Fixed